src/unistd.rs

```diff
@@ -827,7 +827,8 @@ pub fn pipe() -> Result<(RawFd, RawFd)> {
 // libc only defines `pipe2` in `libc::notbsd`.
```

Comment:
Can you clean up this comment? Seems out of date since freebsd support is being added.

Additionally, it appears that this is supported by NetBSD and OpenBSD according to `libc`, so let's add support for bind2 on those platforms as well.

Reply:
I didn't check those platforms.  It looks like pretty much everything supports it now, even Illumos.